front-end/src/assets/styles/login.scss

```diff
@@ -0,0 +1,102 @@
+@import './variables.scss';
+
+.col--left {
+    position: relative;
+    height: 100vh;
+}
+
+.subject-title {
+    margin-top: 15vh;
+}
+
+
+.description-title {
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    text-align: center;
+}
+
+.col-left {
+    position: relative;
+    height: 100vh;
+}
+
+
+
+.login-inner {
+    width: 100%;
+    max-width: 330px;
+    margin: 0 auto;
+
+    .create_account_link {
+        color: #1261A3;
+        font-size: 13px;
+        margin-top: 8px;
+    }
+    .login_account_link {
```

Comment:
login_account_link is used in Login.js
create_account_link is used in CreateAccount.js

Please couple component CSS, js and html together with itself only unless it is shared.

aka
have the CSS used in Login component in Login.scss (or alternatively put it inside Login.js)
have the CSS used in CreateAccount component in CreateAccount.scss (or alternatively put it inside CreateAccount.js)